dev/builddeps-veloxbe.sh

```diff
@@ -29,6 +29,7 @@ VELOX_REPO=""
 VELOX_BRANCH=""
 VELOX_HOME=""
 VELOX_PARAMETER=""
+NUM_THREADS=""
```

Comment:
maybe we need default value here `$(nproc --ignore=2)`

Reply:
> maybe we need default value here `$(nproc --ignore=2)`

sgtm